Pokemon App/Pokemon App/Core/Utils/ObjectMapper.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ObjectMapper.swift
+//  Pokemon App
+//
+//  Created by Mohammad Azri on 21/11/22.
+//
+
+import Foundation
+
+class ObjectMapper {
+    func mapListPokemonResponseToListPokemonDomain(listPokemonResponse: [PokemonResponse]) -> [Pokemon] {
```

Comment:
ini bisa lu bikin static aja, karena lu gabutuh instance dari ObjectMapper class ini cuma sabagai namespace aja. 
```suggestion
    static func mapListPokemonResponseToListPokemonDomain(listPokemonResponse: [PokemonResponse]) -> [Pokemon] {
```

Reply:
gas Bang Kita diskusinya kapan ya?

Reply:
Tolong dicek kembali Bang sudah diperbaiki